docs/uml-class-diagrams/admin/test/GpsSystemTests/GpsSystemTests.puml

```diff
@@ -2,7 +2,21 @@
 
 package edu.oakland.test.admin {
     class GpsSystemTests {
-
+        + void initDisplayGPSInterfaceNotNull()
+        + void initSatelliteNamesNotNull()
+        + void gpsSystemsConstructedSuccessfully()
+        + void useCase1RunSuccessfully()
+        + void uC1ScannerNotNull()
+        + void useCase2RunSuccessfully()
+        + void uC2ScannerNotNull()
```

Comment:
We may need more test cases here for what happens for each connection, but that may play out in integration testing instead... this is good enough for now, just be aware if we make this method more than just a single path, we'll need multiple test cases.